src/cpp/shared/game/game.cpp

```diff
@@ -258,7 +258,7 @@ void game::ProcessOnMapHeroes() {
   HeroExtra *mapExtraHero;
   char isJail;
 
-  memset(heroExists, 0, 54u);// Only 54 out of 56 bytes set to zero, for whatever reason.
+  memset(heroExists, 0, 54u);
```

Comment:
Magic number

Reply:
I guess I got lazy in anticipation of the changes to come to that line. :/
Taking care of that now.